build/com.liferay.ide-repository/bundle/bundle.properties

```diff
@@ -8,18 +8,19 @@ builder.dir=${rootdir}/builder/eclipse
 builder.exe=${builder.dir}/eclipse
 updatesite=file:repository
 all.features=com.liferay.ide.eclipse.tools.feature.group,com.liferay.ide.alloy.feature.group,com.liferay.ide.maven.feature.group,com.liferay.ide.enterprise.feature.group
+update.features=org.eclipse.buildship.feature.group
```

Comment:
What if we add org.eclipse.buildship.feature.group to the all.features group and try installing it there instead?  We want to make sure that people can upgrade from buildship 2.1 to 2.2 using our updatesite and this would test that.